src/embedded_flash/mod.rs

```diff
@@ -47,6 +56,37 @@ mod test {
         };
         Storage::new(store, options)
     }
+
+    fn is_page_aligned(x: usize) -> bool {
+        x & (PAGE_SIZE - 1) == 0
+    }
+
+    /// Mock implementation of a partition, only tests page alignment.
+    pub struct UpgradeLocations;
+
+    impl UpgradeLocations {
```

Comment:
Why is this not a trait?

Reply:
I added a trait, see the general comment.